.github/actions/constellation_recover/action.yml

```diff
@@ -0,0 +1,74 @@
+name: Constellation recover
```

Comment:
Could you move this file to `e2e_recover` ? All payloads for `e2e_test` should start with `e2e_`